python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -98,7 +101,7 @@ def _setNamePath(self, _namePattern='', _localDir=''):
     def _on_attribute__set__(self, obj_type, attrib_name):
         # This is defining the object as uncopyable from outputfiles... do we want this mechanism still?
         r = copy.deepcopy(self)
-        if getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
+        if _getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
```

Comment:
OK, nothing appeared to break, now to run the full set of tests.
Can't do a global import of Job I don't think as Job imports this class by name.